javascript.js

```diff
@@ -0,0 +1,195 @@
+const myArray = [
+  {
+    class: 'background-img1 background-img',
+    class2: 'card-img',
+    subtitle: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    popuptext: 'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.',
+    image: {
+      image1: './imgs/placeholder.png',
+      popup: './imgs/SnapshootPortfolio.png',
+    },
+    langlist: {
+      css: 'css',
+      html: 'html',
+      bootstrap: 'bootstrap',
+      ruby: 'Ruby',
+      lags: 'Ruby on rails',
+    },
+    btn: 'See Project',
+    style: {
+      style1: 'section-1',
+    },
+    button: {
+      live: 'See live',
+      source: 'See source',
+    },
+  },
+  {
+    class: 'background-img2 background-img',
+    class2: 'card-img1',
+    subtitle: 'Multi-Post Stories',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    image: {
+      image1: './imgs/placeholder.png',
+    },
+    langlist: {
```

Comment:
- [ ] kindly centralize the popup in the desktop version as well. great job

Reply:
@tufoinnkuo10 thanks, I will make the changes